packages/wrangler/src/config/index.ts

```diff
@@ -264,16 +264,21 @@ export function printBindings(
 				({ name, class_name, script_name }) => {
 					let value = class_name;
 					if (script_name) {
-						const registryDefinition = context.registry?.[script_name];
-						if (
-							registryDefinition &&
-							registryDefinition.durableObjects.some(
-								(d) => d.className === class_name
-							)
-						) {
-							value += ` (defined in 🟢 ${script_name})`;
+						if (context.local) {
+							const registryDefinition = context.registry?.[script_name];
+
+							if (
+								registryDefinition &&
+								registryDefinition.durableObjects.some(
+									(d) => d.className === class_name
+								)
+							) {
+								value += ` (defined in 🟢 ${script_name})`;
+							} else {
+								value += ` (defined in 🔴 ${script_name})`;
+							}
 						} else {
-							value += ` (defined in 🔴 ${script_name})`;
+							value += ` (defined in ${script_name})`;
```

Comment:
what's the logic based on which we apply these red/green bullets? Cause it seems we apply them in some places but no in others?

Reply:
Before this PR we applied them to all DOs & service bindings based on whether they were in the dev registry. After this PR we apply to all DOs & service bindings based on whether they were in the dev registry, but hide them if not in local dev